pkg/handlers/v2/generic/mutation/controlplanevirtualip/inject.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2023 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package controlplanevirtualip
```

Comment:
This is unchanged from the previous version and is the only handler that was copied over.